### PR TITLE
Dev hannah

### PR DESCRIPTION
**Pull request by hannbus**

### CODE REVIEW

**Comment by Kohulan** [comment]
Looks ok to me.

File: CIDER/cider.py [hunk 1]
```diff
@@ -60,6 +61,7 @@
 from fpdf import XPos
 from fpdf.enums import Align
 from datetime import date
+import datetime
```
Comment (JonasSchaub) [inline CIDER/cider.py:64]: You have one specific import from datetime (date) here and then you import the complete package in the next line. Please, either remove the specific import in the first line or turn the second import also into a specific one (preferred option).

**Review by JonasSchaub — CHANGES_REQUESTED** [review]

**Comment by JonasSchaub** [comment]
Other than that, it is all good!

File: CIDER/cider.py (hunk 1 above)
Comment (hannbus) [reply CIDER/cider.py]: Ok, I changed that.

**Comment by JonasSchaub** [comment]
Sorry, @hannbus , but "from datetime import datetime" does not make much sense. Can't you do "from datetime import now" or something similar?

**Comment by OBrink** [comment]
@JonasSchaub, `from datetime import datetime` is perfectly fine. `now()` is a class method of a class called `datetime` which is imported from the module `datetime`. Admittedly, this is a bit odd

**Comment by Kohulan** [comment]
More info : https://docs.python.org/3/library/datetime.html#datetime.datetime
So you can basically do this: 
```
from datetime import date, datetime
```

**Comment by JonasSchaub** [comment]
Okokok, I apologise for my very limited Python knowledge! I would recommend doin it in one import statement as Kohulan suggested, then.

**Pull request merged**